src/themes/shared/tokens/aspectRatios.ts

```diff
@@ -0,0 +1,22 @@
+import { defineTokens } from '@chakra-ui/react';
+
+export const aspectRatios = defineTokens.aspectRatios({
```

Comment:
Where do they originate?

Reply:
Those are chakra defaults

Reply:
Maybe align with UX if we wanna restrict them or they are good as is